src/client/player.h

```diff
@@ -2,11 +2,17 @@
 
 #include <stdbool.h>
 #include <SDL.h>
+#include "main.h";
+#include "events.h";
+
 
 typedef struct
 {
     SDL_Rect rect;
     bool infected;
+    Mix_Chunk *walking;
```

Comment:
Please don't have every instance of Player struct contain a loaded music
These look unused anyway